assets/src/domain/eventEditor/ui/tickets/useAddTicketModal.ts

```diff
@@ -0,0 +1,36 @@
+import { useCallback } from 'react';
+import { __ } from '@wordpress/i18n';
+import { FormProps } from 'react-final-form';
+
+import useTicketFormConfig from './ticketForm/useTicketFormConfig';
+import { useEditorModal, EditorModal, ModalClose } from '@appLayout/editorModal';
+import { useTicketMutator, CreateTicketInput } from '@edtrServices/apollo/mutations';
+import useEvent from '@edtrServices/apollo/queries/events/useEvent';
+
+const useAddTicketModal: EditorModal = () => {
+	const { createEntity } = useTicketMutator();
+	const { closeEditor } = useEditorModal();
+
+	const { name: eventName = '' } = useEvent() || {};
+
+	const onClose = useCallback<ModalClose>((): void => {
```

Comment:
If this callback isn't passing any props maybe we can simplify this and avoid callback?